code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -422,6 +422,75 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	else
 		return ..()
 
+//////////////
+//FANCY CIGS//
+//////////////
+
+/obj/item/clothing/mask/cigarette/bidi
```

Comment:
It's kind of a shame that these don't have a couple unique reagents in each, but then again it's not like we have flavors